src/vsl/parser/nodes/typeAlias.js

```diff
@@ -1,27 +0,0 @@
-import Node from './node';
```

Comment:
I found the production rule for `TypeAlias` but it wasn't referenced anywhere so I just deleted it.